recipes/boost/all/conanfile.py

```diff
@@ -651,14 +651,14 @@ def requirements(self):
         if self._with_bzip2:
             self.requires("bzip2/1.0.8")
         if self._with_lzma:
-            self.requires("xz_utils/5.4.4")
+            self.requires("xz_utils/5.4.5")
```

Comment:
+1 for using `5.4.5`, which is the version used in nearly all other recipes except ones still awaiting migration to Conan 2.0.